internal/configs/sharding/mainnet.go

```diff
@@ -26,6 +26,7 @@ const (
 	mainnetV1_1Epoch = 19
 	mainnetV1_2Epoch = 25
 	mainnetV1_3Epoch = 36
+	mainnetV1_4Epoch = 42
```

Comment:
shard1 is already on 42, it can't be 42. It must be something next Wednesday/Thursday.